extension/demo/src/bg/background.js

```diff
@@ -24,62 +31,89 @@ function onPopupOpenApps () {
   })
 }
 
-async function onToken (data) {
-  const { accessToken } = data
+function onSetEmitStatus (data) {
```

Comment:
What does emit status mean?

Reply:
Yeah, the name is a little tricky. But it controls whether the payload is sent or not.

Ideas for other names are welcome.

`emitPayload`? `sendPayload`? `shouldSendPayload`?